### DISCUSSION TITLE
Telegram notifications

### DISCUSSION THREAD

**Discussion by javikobcn** [Flexget/Flexget#3049] [Q&A]
Hello, I am trying to edit a config so that when a task fails it sends me a notification via telegram with the name of the file that has failed, would this be possible?

This is my config:

 

> web_server:
> 
>   port: 5050
> 
> schedules:
>   - tasks: '*'
>     interval:
>       minutes: 15
> 
> templates:
> 
>   movies-metainfo:
>     parsing:
>       movie: guessit
> 
>   tv:
>     series:
>       settings:
>         tv:
>           identified_by: ep
> 
>       tv:
>         - Coyote
>         - Ataque a los titanes
>         - Falcon y el Soldado de Invierno
>         - Futurama
>         - Gangs of London
>         - Ozark
>         - The Mandalorian
>         - Chernobyl
> 
>   telegram:
>     notify:
>       entries:
>         message: |+
>           {%if task in ["move movies"]%}*Nueva PelÃ­Â­cula renombrada:* _{{title}}_{%endif%}
> 
>           *TÃ­Â­tulo:* {%if tmdb_name|default(False)%}{{tmdb_name}} ({{tmdb_year}}){%else%}PelÃ­Â­cula no encontrada en tmdb{%endif%} 
>           *TamaÃ±o:* {{content_size}} MB
>           *ValoraciÃ³n:* {{tmdb_rating|d('?')}}/10 - {{tmdb_popularity|d('N/A')}} votos
>           [Poster]{% if tmdb_posters|default(False) %}({{tmdb_posters[0]|replace("_", "%5F")}}){% endif %}
>         via:
>           - telegram:
>               bot_token: ******
>               parse_mode: markdown
>               recipients:
>                 - group: *****
> 
>   telegram2:
>     notify:
>       entries:
>         message: |+
>           {%if task in ["sort-tv"]%}*Nuevo CapÃ­Â­tulo renombrado:* _{{title}}_{%endif%}
> 
>           *TÃ­tulo:* {{ series_id }}{% if tvdb_ep_name|default(False) %} - {{ tvdb_ep_name|replace('/', '_')|replace(':', '-')|replace(',', '') }}{%else%}Serie no encontrada en tvdb{%endif%} 
>           *TamaÃ±o:* {{content_size}} MB
>           *ValoraciÃ³n:* {{tvdb_rating|d('?')}}/10
>           [Poster]{% if tvdb_posters|default(False) %}({{tvdb_posters[0]|replace("_", "%5F")}}){% endif %}
>         via:
>           - telegram:
>               bot_token: ******
>               parse_mode: markdown
>               recipients:
>                 - group: ****
> 
> tasks:
>   
>   move movies:
>     priority: 3
>     template:
>       - movies-metainfo
>       - telegram
>     filesystem:
>       path: /downloads/
>       recursive: yes
>       retrieve: files
>       regexp: '.*\.(avi|mkv|mp4)$'
>     seen: local
>     regexp:
>       reject:
>         - \b(duo|tri|quadri|tetra|penta)logy\b: {from: title}
>         - s\d{2}(e\d{2,})?: {from: title} 
>     require_field: 
>       - tmdb_name
>       - movie_name
>     accept_all: yes
>     tmdb_lookup:
>       language: es
>     move:
>       to: "/media/Peliculas/"
>       rename: "{{tmdb_name|replace('/', '_')|replace(':', ' -')|replace(',', '') }}"
>       along:
>         extensions:
>           - sub
>           - srt
>         subdirs:
>           - Subs
>       clean_source: 50  
>  
>   sort-tv:
>     priority: 2
>     filesystem:
>       path: /downloads/
>       regexp: '.*\.(avi|mkv|mp4)$'
>       recursive: yes
>     seen: local
>     template:
>       - tv
>       - telegram2
>     series:
>       settings:
>         agroup:
>           parse_only: yes
>     regexp:
>       reject:
>         - sample
>     require_field:
>       - series_name
>       - series_season
>     all_series:
>       parse_only: yes
>     accept_all: yes          
>     thetvdb_lookup:
>       language: es
>     move:
>       # Folder to move to
>       to: >
>         /media/Series/{{ tvdb_series_name|default(series_name)|replace('/', '_')|replace(':', '-')|replace(',', '.') }}/{% if series_id_type == 'ep' %}Temporada {{ series_season|pad(2) }}/{% endif %}
>       # Filename (inside of folder) to move to
>       rename: >
>         {{ series_id }}{% if tvdb_ep_name|default(False) %} - {{ tvdb_ep_name|replace('/', '_')|replace(':', '-')|replace(',', '') }}{% endif %}.{{ location[-3:] }}
>       along:
>         extensions:
>           - sub
>           - srt
>         subdirs:
>           - Subs
>       clean_source: 7000
>     if:
>       - movedone:
>           set:
>             path: '{{ movedone }}'
> 
>   decompress-rar:
>     priority: 1
>     accept_all: yes
>     filesystem:
>       path: /downloads/
>       recursive: yes
>       regexp: '.*\.(rar|r0+[01](/01)|zip)$'
>     decompress:
>       to: "/downloads/"
>       keep_dirs: no
>       delete_archive: yes

**Comment by Vctrsnts** [comment]
Hi.

You wanted the solution??

Thanks
